ofl/notosansnagmundari/METADATA.pb

```diff
@@ -0,0 +1,27 @@
+name: "Noto Sans Nag Mundari"
+designer: "Google"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2023-03-16"
+fonts {
+  name: "Noto Sans Nag Mundari"
+  style: "normal"
+  weight: 400
+  filename: "NotoSansNagMundari[wght].ttf"
+  post_script_name: "NotoSansNagMundari-Regular"
+  full_name: "Noto Sans Nag Mundari Regular"
+  copyright: "\"Copyright 2023 The Noto Project Authors (https://github.com/notofonts/nag-mundari)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
```

Comment:
@simoncozens missing subset, primary script, languages

Reply:
Nag Mundari subset was not detected because we don't have one. I'll make one...